packages/webviz-core/src/components/PlayerManager.js

```diff
@@ -66,12 +65,17 @@ function PlayerManager({ importPanelLayout, children, userNodes, setNodeDiagnost
   React.useEffect(
     () => {
       const params = new URLSearchParams(window.location.search);
-      console.log(remoteBagConnectionURL);
-      const remoteBagBaseURL = remoteBagConnectionURL;
-      const remoteDemoBagUrl = "https://open-source-webviz-ui.s3.amazonaws.com/demo.bag";
+      const remoteDemoBagUrl =
+        "https://storage.enway.ai:9000/bags-studio-16/studio_16_01.bag?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20191126%2F%2Fs3%2Faws4_request&X-Amz-Date=20191126T123246Z&X-Amz-Expires=432000&X-Amz-SignedHeaders=host&X-Amz-Signature=336681d16d39b0100893a6eaf26c7b3755feedbe5110f8335810cde27511eb16";
```

Comment:
https lives at port 443 and has valid Let's Encrypt certificates

Reply:
need to add storage.enway.ai to alt_names

Reply:
done! updated to new url.